Hard-code source term for code generation, until SymPy error is found.

tkphd opened the pull request:
**Hard-code source term until SymPy bug is resolved **

Closes #756.

Automatic generation of the source term for Benchmark 7 using SymPy produces an erroneous expression. Root cause is as yet undetermined, so this PR applies a stopgap to let benchmarkers move forward with less confusion.

<!--
  Text below provides an easy link for PR reviewers to click.
  After submitting your request, change the `[live-site]:` URL
  at the end with the actual `{pull-request-number}` assigned
  by GitHub, without the '#'.
-->

These changes can be [viewed live][live-site].

**Remember to tear down the [live site][live-site] when merging
or closing this pull request.**

[live-site]: http://random-cat-757.surge.sh

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/usnistgov/pfhub/757)
<!-- Reviewable:end -->

stvdwtt commented:
I'm going to check this by the end of the day.

wd15 commented:
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[benchmarks/benchmark7.ipynb, line 61 at r1](https://reviewable.io:443/reviews/usnistgov/pfhub/757#-L5eZ3mORYo0nnPtvl98:-L5eZ3mORYo0nnPtvl99:b-acwpn) ([raw file](https://github.com/usnistgov/pfhub/blob/a69be881a9a72b1918e66085d60844352e23c7a7/benchmarks/benchmark7.ipynb#L61)):*
> ```
>      "metadata": {},
>      "output_type": "execute_result"
>     }
> ```

The output for this cell has disappeared, which means that the "code toggle" button is missing. So, we can't switch between "with code" and "without code" viewing modes on the front end, see http://random-cat-757.surge.sh/benchmarks/benchmark7.ipynb/. If you keep this output then we can switch between the two modes, see https://pages.nist.gov/pfhub/benchmarks/benchmark7.ipynb/ 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/usnistgov/pfhub/757)*
<!-- Sent from Reviewable.io -->

wd15 commented:
Also, the equation numbers appear to have gone in http://random-cat-757.surge.sh/benchmarks/benchmark7.ipynb/, but are in https://pages.nist.gov/pfhub/benchmarks/benchmark7.ipynb/.

tkphd commented:
Good catch -- not sure why that cell output dropped. Fixed now, hopefully...

tkphd commented:
Yep, those fancy characters are definitely syntax fails. I'll fix this afternoon.

tkphd commented:
Fixed, I hope. I removed the `S_part` expressions and left only the monolithic source terms. This clarifies which code to copy, but makes parsing `S` by eye more difficult.

I've tested this code with CUDA C, and it seems to work. I can't speak to the other languages either; they're included for inclusiveness, but may require editing to be useful.
